applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -35,47 +37,50 @@ Buttons
-+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-| **BTN 5**     | Long-pressed during startup: Clears the previously stored bonding information.                            | * :ref:`nrf53_audio_unicast_server_app`     |
-|               |                                                                                                           | * :ref:`nrf53_audio_unicast_client_app`     |
-|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|               | Pressed during playback: Mutes the playback volume.                                                       | * :ref:`nrf53_audio_unicast_server_app`     |
-|               |                                                                                                           | * :ref:`nrf53_audio_unicast_client_app`     |
-|               +-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-|               | Pressed on a BIS headset during playback: Change the gateway, if more than one is available.              | :ref:`nrf53_audio_broadcast_sink_app`       |
-+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
-| **RESET**     | Resets the device to the originally programmed settings.                                                  | All                                         |
-|               | This reverts any changes made during testing, for example the channel switches with **VOL** buttons.      |                                             |
-+---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+
++---------------+-----------------------------------------------------------------------------------------------------------+---------------------------------------------+-------------------------------+
+| Button        | Function                                                                                                  | Applications                                | Development Kit               |
+|               |                                                                                                           |                                             +------------------+------------+
+|               |                                                                                                           |                                             | nRF5340 Audio DK | nRF5340 DK |
```

Comment:
This addition is not matching the list of supported boards in Requirements.

Reply:
As above.